src/main/java/com/cryptomorin/xseries/XItemStack.java

```diff
@@ -226,8 +239,43 @@ public static void serialize(@Nonnull ItemStack item, @Nonnull ConfigurationSect
                 fwc.set("fade-colors", fadeColors);
                 i++;
             }
-            //} else if (meta instanceof MapMeta) {
-        } else if (XMaterial.supports(14)) {
+        } else if(meta instanceof BookMeta) {
+            BookMeta book = (BookMeta)meta;
+            ConfigurationSection info = config.createSection("information");
+            info.set("title", book.getTitle());
+            info.set("author", book.getAuthor());
+            if(XMaterial.supports(9)) {
+                BookMeta.Generation generation = book.getGeneration();
+                if (generation != null) {
+                    info.set("generation", book.getGeneration().toString());
+                }
+            }
+            info.set("pages", book.getPages());
+        } else if(meta instanceof MapMeta) {
+            MapMeta map = (MapMeta)meta;
+            ConfigurationSection info = config.createSection("information");
+            info.set("scaling",map.isScaling());
```

Comment:
Same as above ^

Reply:
This is how it looks with the organization i'm proposing.
![organization](https://user-images.githubusercontent.com/18015599/109751974-8f612880-7bad-11eb-9b15-c454369d63b3.png)

Reply:
`information` -> `map`
The `view` section is redundant imo. Just move it all to `map`
I think it'd be more convenient to use `center: <x>, <y>` format, but since that's not a common pattern in the config, it'd be better to just move them to their own section to avoid confusion.
```yml
center:
  x: 1
  y: 0
```

Reply:
The MapView is another object taken from MapMeta class, don't you think is good to separate them?

Reply:
Well, sure. Maybe add `setColor` and `setLocationName` support as well. [MapMeta](https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/inventory/meta/MapMeta)

Reply:
Perfect.